spec/controllers/presentations_controller_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'rails_helper'
+
+RSpec.describe PresentationsController, type: :controller do
+  describe 'GET #new' do
+    subject { get :new }
+
+    it { is_expected.to render_template :new }
+    it { is_expected.to be_successful }
+  end
+
+  describe 'GET #show' do
+    subject { get :show, params: { id: 1 } }
+    let!(:presentation) { FactoryGirl.create(:presentation) }
```

Comment:
Instead of sending in `params: { id: 1 }` I think it would be better to send in `params: { id: presentation.id }`